requirements.txt

```diff
@@ -4,5 +4,5 @@ Fabric==0.9.0
 nose==1.0.0
 rdflib==3.2.1
 argparse==1.1
--e git://github.com/mattbasta/fastchardet#egg=fastchardet
+fastchardet
```

Comment:
It's also good. Amending or squashing will always need a force push. I can do the squashing myself when merging, which i'm going to do know. Thanks!